src/apify_client/clients/resource_clients/task.py

```diff
@@ -0,0 +1,214 @@
+from typing import Any, Dict, List, Optional, cast
+
+from ..._consts import ActorJobStatus
+from ..._errors import ApifyApiError, ApifyClientError
+from ..._utils import _catch_not_found_or_throw, _encode_json_to_base64, _filter_out_none_values_recursively, _parse_date_fields, _pluck_data
+from ..base.resource_client import ResourceClient
+from .webhook_collection import WebhookCollectionClient
+
+
+class TaskClient(ResourceClient):
+    """Sub-client for manipulating a single task."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the TaskClient."""
+        super().__init__(*args, resource_path='actor-tasks', **kwargs)
+
+    def get(self) -> Optional[Dict]:
+        """Retrieve the task.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-object/get-task
+
+        Returns:
+            The retrieved task
+        """
+        return self._get()
+
+    def update(
+        self,
+        *,
+        name: Optional[str] = None,
+        task_input: Optional[Dict] = None,
+        build: Optional[str] = None,
+        memory_mb: Optional[int] = None,
+        timeout_secs: Optional[int] = None,
+    ) -> Dict:
+        """Update the task with specified fields.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-object/update-task
+
+        Args:
+            name (string, optional): Name of the task
+            build (str, optional): Specifies the actor build to run. It can be either a build tag or build number.
```

Comment:
All the comments around are worded as a definition (e.g. "Name of the task"), while this one seems more like an "explanation" (begins with a verb).